src/elements/radio-button/common/radio-button-common.scss

```diff
@@ -57,6 +57,10 @@
   }
 }
 
+:host([size='xs']) {
+  --sbb-radio-button-dimension: #{sbb.px-to-rem-build(20)};
```

Comment:
I wonder whether we should add an 'xs' size token for [icons](https://github.com/sbb-design-systems/lyne-design-tokens/blob/main/designTokens/size.ts#L193) (we have other cases where 20px was the size for icon-like elements)